src/Agent.Sdk/ProcessInvoker.cs

```diff
@@ -63,14 +63,19 @@ public void Reset()
         }
 
         public bool DisableWorkerCommands { get; set; }
+        public bool TryUseGracefulShutdown { get; set; }
+        public TimeSpan SigintTimeout { get; set; }
```

Comment:
Shouldn't be readonly?

Reply:
I made it mutable to avoid the Invoker re-creation inside the ProcessInvokerWrapper if we want to change some parameters